ini/ini_map.ts

```diff
@@ -88,25 +80,25 @@ export class IniMap {
       }
       return false;
     },
-    getAtLine: (line: number): string | undefined => {
+    getTextAtLine: (line: number): string | undefined => {
```

Comment:
Recommend naming these with `Text` in the name in order to possibly allow having a `getAtLine` in the future that returns a comment object.